ebill_paynet/tests/test_invoice_message_b2b.py

```diff
@@ -108,8 +108,8 @@ def test_invoice_qr_rounding(self, get_ic_ref):
         invoice.action_post()
         invoice.payment_reference = "1234567890"
         invoice.partner_bank_id = self.partner_bank.id
-        # amount inside is not rounded
-        self.assertEqual(invoice.amount_by_group[0][1], 0.35000000000000003)
+        # amount inside is not rounded but the field goes 4 decimals deep
+        self.assertAlmostEqual(invoice.amount_by_group[0][1], 0.35000000000000003, 4)
```

Comment:
interesting why this test starts to fail current master branch seems ok, addon doesn't introduce related changes

Reply:
@i-vyshnevska 
Sorry for a beginer question, is there anthing to do yet on the PR?

Reply:
> @i-vyshnevska Sorry for a beginer question, is there anthing to do yet on the PR?

I'm wondering the same thing

Reply:
@i-vyshnevska do you thing is it better to close this PR or does it still needs changes?